src/Traits/Util.php

```diff
@@ -42,4 +42,26 @@ public function launchAppiumApp()
 
         return $response;
     }
+
+    /**
+     * DESC
```

Comment:
@anooptajawal Put the reference and also the description of what this function will do